packages/kit/src/runtime/client/renderer.js

```diff
@@ -718,7 +718,7 @@ export class Renderer {
 			if (cached) return cached;
 		}
 
-		const [pattern, a, b, get_params, has_shadow] = route;
+		const [pattern, a, b, get_params, has_shadow, routeKey] = route;
```

Comment:
removes the `has_shadow` flag, per comment above — would need corresponding change on line 772. note that we use snake_case, not camelCase

```suggestion
		const [pattern, a, b, get_params, shadow_key] = route;
```